.rhdh/bundle/manifests/rhdh-operator.csv.yaml

```diff
@@ -62,6 +62,8 @@ spec:
 
     System Architects can benefit by implementing a tailored platform with a complementary suite of verified and curated tools and components needed for operations teams to support developers—within a centralized, consistent location. Development teams can experience increased productivity, fewer development team obstacles, and simplified governance of technology choices with self-service and guardrails.
 
+    The telemetry data collection feature is enabled by default. Red Hat Developer Hub sends telemetry data to Red Hat by using the `backstage-plugin-analytics-provider-segment` plugin. To disable this and to learn what data is being collected, see https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.2/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-rhdh-telemetry_admin-rhdh
```

Comment:
since the version is hardcoded here, we will need to make sure that this version is updated at intervals when we branch for 1.2.x and main branch moves to 1.3. See https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/scripts/tagRelease.sh#L317-322

Reply:
Good point. Just curious, is it possible to link to a kind of "latest" link of the doc?

Reply:
nope, there's no "latest" deep doc linking. https://access.redhat.com/documentation/en-us/red_hat_developer_hub/ will bounce to the latest release https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1 but that won't get you to the right spot :(

Reply:
@04kash might want to add something for this into your MR https://gitlab.cee.redhat.com/rhidp/rhdh/-/merge_requests/21 (or else a second MR)